src/Logging/Logging.Testing/src/LoggedTest/LoggedTestBase.cs

```diff
@@ -102,5 +103,19 @@ public virtual void Dispose()
             _initializationException?.Throw();
             _testLog.Dispose();
         }
+
+        Task ITestMethodLifecycle.OnTestStartAsync(TestContext context, CancellationToken cancellationToken)
```

Comment:
Can I use OnTestStartAsync to conditionally skip a test?
https://github.com/aspnet/AspNetCore/issues/11301

Reply:
Unfortunately no. I didn't try to build control-flow into these methods.